src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,65 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import subway.station.StationRepository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    StationAcceptanceTest stationAcceptanceTest = new StationAcceptanceTest();
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     * */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+        stationAcceptanceTest.createStationByName("사당역");
```

Comment:
```suggestion
        stationAcceptanceTest.지하철_역_추가_요청("사당역");
```

`StationAcceptanceTest` 의 `지하철_역_추가_요청` 메서드를 public 으로 변경 후 사용하는 것이 더 좋을 것 같아요 😅 
`@Test` 를 붙인 메서드 자체를 쓰는 것보다 필요한 부분만 가져가서 쓰는 것이 좋습니다